_site/_sass/color_schemes/light.scss

```diff
@@ -0,0 +1,22 @@
+$sidebar-color: #43a047;
+$border-color: #43a047;
```

Comment:
nit: feel free to ignore

I think this border color causes all the code-blocks to also get the green border. IMHO this interrupts with fast reading. 

Compare [generated action-cache](https://luxe.github.io/buildfarm/docs/architecture/action_cache/) with [wiki action-cache](https://github.com/bazelbuild/bazel-buildfarm/wiki/ActionCache)

Reply:
haha yeah that's too much green. we can give get rid of the code border or make it black

Reply:
fixed